.github/ISSUE_TEMPLATE/proposed-change-request.md

```diff
@@ -2,6 +2,9 @@
 name: Proposed Change Request
 about: This template should be used as applicable for any proposed change to the CF
   convention.
```

Comment:
Line break between CF and convention.

Reply:
That's not actually a problem since in markdown, line breaks don't render...

Reply:
Yes, I knew that, but since you're also cleaning spaces at the end of the line in this PR, I thought you would appreciate cleaning this as well. Nevermind.

Reply:
Huh -- I didn't mean to touch this file! Was using the github editor online and don't think I ever opened it. 🤷‍♂